frontend/src/concepts/pipelines/content/pipelineSelector/PipelineSelectorTableRow.tsx

```diff
@@ -35,13 +35,7 @@ const PipelineSelectorTableRow: React.FC<PipelineSelectorTableRowProps> = ({ obj
         }
         triggerRef={tooltipRef}
       />
-
-      <Tr
-        ref={tooltipRef}
-        onRowClick={onClick}
-        isClickable
-        data-id="pipeline-selector-table-list-row"
```

Comment:
why did you drop a data id?

Reply:
its not used anywhere in tests. also im pretty sure tests only look for data-testid?

Reply:
Right, just some small cleanups since I already changed this file.

Reply:
ok. marking this resolved then